src/operand/reference.ts

```diff
@@ -27,14 +42,27 @@ function contextValueLookup (ctx: Context, key: string): Result {
   let pointer = ctx
 
   for (let i = 0; i < keys.length; i++) {
```

Comment:
Instead of this can we use lodash `get` method?

Reply:
well, we're doing that resolution of inner references, etc. but potentially yes / maybe

Reply:
Could work... do we want to add lodash as a dependency?

Reply:
I would say, not keep the library as pure as possible, unless there is some signification benefit of using the lodash get